Jenkinsfile

```diff
@@ -40,10 +40,10 @@ def createBuildTask(jvmsToTest,currentVersion) {
 			withEnv(["JVMS_TO_TEST=${jvmsToTest}"]) {
 				checkout scm
 
-				gradlew "-Pversion=${currentVersion} clean assemble compileTestJava --parallel"
```

Comment:
Keep the previous name of the property, as it will directly set the version of the project. No more default handling needed in the build.gradle file.

Reply:
my initial thought was, that maybe the version property could clash with the version itself. therefore i renamed it... but this is for sure a better way

Reply:
Absolutely, it clashes, but in a good way :D